stdlib/socketserver.pyi

```diff
@@ -39,50 +41,25 @@ class BaseServer:
     def close_request(self, request: _RequestType) -> None: ...  # undocumented
 
 class TCPServer(BaseServer):
+    allow_reuse_port: bool
+    request_queue_size: int
     def __init__(
-        self,
-        server_address: tuple[str, int],
-        RequestHandlerClass: Callable[..., BaseRequestHandler],
-        bind_and_activate: bool = ...,
+        self, server_address: tuple[str, int], RequestHandlerClass: _RequestHandlerClassCallable, bind_and_activate: bool = ...
     ) -> None: ...
     def get_request(self) -> tuple[_socket, Any]: ...
-    def finish_request(self, request: _RequestType, client_address: _AddressType) -> None: ...
-    def handle_error(self, request: _RequestType, client_address: _AddressType) -> None: ...
-    def process_request(self, request: _RequestType, client_address: _AddressType) -> None: ...
-    def verify_request(self, request: _RequestType, client_address: _AddressType) -> bool: ...
-    def shutdown_request(self, request: _RequestType) -> None: ...  # undocumented
-    def close_request(self, request: _RequestType) -> None: ...  # undocumented
```

Comment:
In the past, we have kept overrides that don't change the signature but exist in the implementation, although nobody seems to remember why.

Reply:
It would be good to establish a general policy on this point — several of my PRs deleting duplicate method redefinitions have been accepted:

- https://github.com/python/typeshed/pull/6630
- https://github.com/python/typeshed/pull/6545

Reply:
I'm OK with removing them.